process-migration-service/frontend/cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
```

Comment:
I would remove this file if not planned to use - I mean in tests you could put here a pre-exported migration plan and test importing back. But this example.json is useless.

Reply:
ok